certif/app/controllers/authenticated/sessions/new.js

```diff
@@ -28,10 +53,28 @@ export default class SessionsNewController extends Controller {
   @action
   onDatePicked(selectedDates, lastSelectedDateFormatted) {
     this.session.date = lastSelectedDateFormatted;
+    this.isSessionDateMissing = false;
   }
 
   @action
   onTimePicked(selectedTimes, lastSelectedTimeFormatted) {
     this.session.time = lastSelectedTimeFormatted;
+    this.isSessionTimeMissing = false;
+  }
+
+  areSessionFieldsMissing() {
```

Comment:
```suggestion
  checkSessionFieldsMissing() {
```
On modifie des valeurs ici, le "are" suppose que c'est juste de la verification de donnée (le check aussi mais un peu moins)